### PR TITLE
ENH: ShadowServer's Radmin & CAIDA feeds

### PR DESCRIPTION
**Pull request by sinus-x**
Add feed for ShadowServer's [Accessible Radmin](https://www.shadowserver.org/what-we-do/network-reporting/accessible-radmin-report/)
Add feed for ShadowServer's [CAIDA IP Spoofer](https://www.shadowserver.org/what-we-do/network-reporting/caida-ip-spoofer-report/)

### CODE REVIEW

**Comment by wagner-certat** [comment]
We receive the Radmin feed, but I couldn't find the CAIDA IP Spoofer feed in our inbox (I searched for CAIDA as well as for Spoofer). @th-certbund do you receive this one?

File: intelmq/bots/parsers/shadowserver/config.py [hunk 1]
```diff
@@ -2298,6 +2298,66 @@ def convert_date(value):
...
 }
 
+# https://www.shadowserver.org/what-we-do/network-reporting/accessible-radmin-report/
+accessible_radmin = {
+    'required_fields': [
+        ('time.source', 'timestamp', add_UTC_to_timestamp),
+        ('source.ip', 'ip'),
+        ('source.port', 'port', convert_int),
+    ],
+    'optional_fields': [
```
Comment (wagner-certat) [inline intelmq/bots/parsers/shadowserver/config.py:2308]: I'm missing the `version` field here. Value is mostly `Radmin v3.X Radmin Authentication`

**Review by wagner-certat — CHANGES_REQUESTED** [review]
Thank you for this contribution!

I currently only have a small comment on the Radmin feed. As I don't have access to a CAIDA IP Spoofer report myself currently, I postpone the review of that one until I get a report somewhere. (If it turns out that it lasts longer, I will merge the Radmin commit before)

**Comment by wagner-certat** [comment]
P.S.: The most recent CAIDA IP Spoofer Report I have is 2020-07-19, so the question is only if these feed is still "active"

File: intelmq/bots/parsers/shadowserver/config.py [hunk 2]
```diff
@@ -2298,6 +2298,66 @@ def convert_date(value):
...
+        ('source.geolocation.city', 'city'),
+        ('source.reverse_dns', 'hostname', validate_to_none),
+        ('extra.', 'type', validate_to_none),
+        ('extra.', 'naics', convert_int),
+        ('extra.', 'sic', convert_int),
+        ('extra.', 'sector', validate_to_none),
+        # FIXME Is is mappable to some classification.* field? Not included in example data.
+        ('extra.', 'family', validate_to_none),
+        ('source.network', 'network', validate_to_none),
+        ('extra.', 'version', validate_to_none),
```
Comment (wagner-certat) [inline intelmq/bots/parsers/shadowserver/config.py:2348]: That's the IP version which is obvious from the address and network, maybe we can just drop it.

**Comment by JakubOnderka** [comment]
> P.S.: The most recent CAIDA IP Spoofer Report I have is 2020-07-19, so the question is only if these feed is still "active"

Our last report is from 2020-08-21, so it is still active.

**Comment by wagner-certat** [comment]
> > P.S.: The most recent CAIDA IP Spoofer Report I have is 2020-07-19, so the question is only if these feed is still "active"
> 
> Our last report is from 2020-08-21, so it is still active.

Thanks for checking!

File: intelmq/bots/parsers/shadowserver/config.py (hunk 2 above)
Comment (wagner-certat) [reply intelmq/bots/parsers/shadowserver/config.py:2348]: It does, but I had to look at the code myself too :)
Using `False` as field name:
https://github.com/certtools/intelmq/blob/6a657269b0e3ac7518d6e7b8e49535ee154e36b9/intelmq/bots/parsers/shadowserver/parser.py#L184-L187
I will add this to the module-docstring in `config.py`.

File: intelmq/bots/parsers/shadowserver/config.py (hunk 1 above)
Comment (wagner-certat) [reply intelmq/bots/parsers/shadowserver/config.py:2308]: of course, thanks

**Comment by wagner-certat** [comment]
I'm unsure about the classification, I'll try to clarify that. As that may take some time, this PR is hereby merged and scheduled for the next bugfix release :)

As the filename for the caida feed is not `scan_caida` but `caida_ip_spoofer` I adapted the mappings and file names.

Thanks @sinus-x @JakubOnderka 

Follow-up commits: 96ff28b2e65c1912f0f86c68a60af76e89b0e45f 46ef34048ee037d1313bcbdfea1d059c03ffc480

**Pull request merged**